src/main/java/gregtech/common/gui/widget/craftingstation/ItemListGridWidget.java

```diff
@@ -192,14 +192,14 @@ public void readUpdateInfo(int id, PacketBuffer buffer) {
             try {
                 int itemsRemoved = buffer.readVarInt();
                 for (int i = 0; i < itemsRemoved; i++) {
-                    ItemStackKey itemStack = KeySharedStack.getRegisteredStack(buffer.readItemStack());
-                    this.displayItemList.removeIf(it -> it.getItemStackKey().equals(itemStack));
+                    ItemStack itemStack = buffer.readItemStack();
+                    this.displayItemList.removeIf(it -> it.getItemStack().equals(itemStack));
```

Comment:
This doesn't seem correct, since `itemStack.equals(itemStack)` is not desired for use